js/app.js

```diff
@@ -32,4 +32,18 @@ $(function() {
 	if ($nav.length) {
 		createStickyNav($nav, window.requestAnimationFrame)
 	}
+
+	var body = document.body,
+    timer;
+
+	window.addEventListener('scroll', function() {
```

Comment:
In my experience this doesn't actually work. Adding and removing the class causes style recalc and on very large pages can kill the frame budget.